### PR TITLE
Add delete sticker from sticker pack method

### PR DESCRIPTION
**Pull request by denieryd**
# Description

Add delete sticker from sticker pack method

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: botx/client/stickers_api/delete_sticker.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        sticker_pack_id = payload.sticker_pack_id
+        sticker_id = payload.sticker_id
+        path = f"/api/v3/botx/stickers/packs/{sticker_pack_id}/stickers/{sticker_id}"
+
+        json = payload.jsonable_dict()
+
+        response = await self._botx_method_call(
+            "POST",
+            self._build_url(path),
+            json=json,
```
Comment (Nimond) [inline botx/client/stickers_api/delete_sticker.py:43]: `json=payload.jsonable_dict()`

File: botx/bot/bot.py [hunk 2]
```diff
@@ -1080,6 +1084,33 @@ async def add_sticker(
...
+        *,
+        bot_id: UUID,
+        sticker_pack_id: UUID,
+        sticker_id: UUID,
+    ) -> None:
+        """Delete sticker from sticker pack.
+
+        Args:
+            bot_id: Bot which should perform the request.
+            sticker_pack_id: Sticker pack id to indicate to delete from.
```
Comment (lxmnk) [inline botx/bot/bot.py:1098]: Target sticker pack id.

File: botx/client/stickers_api/delete_sticker.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        payload: BotXAPIDeleteStickerRequestPayload,
+    ) -> None:
+        sticker_pack_id = payload.sticker_pack_id
+        sticker_id = payload.sticker_id
+        path = f"/api/v3/botx/stickers/packs/{sticker_pack_id}/stickers/{sticker_id}"
+
+        response = await self._botx_method_call(
+            "POST",
+            self._build_url(path),
+            json=payload.jsonable_dict(),
```
Comment (Nimond) [inline botx/client/stickers_api/delete_sticker.py:41]: Wait. There are no json payload for this method. All goes in query params

**Review by Nimond — CHANGES_REQUESTED** [review]

File: botx/client/stickers_api/delete_sticker.py (hunk 3 above)
Comment (denieryd) [reply botx/client/stickers_api/delete_sticker.py:41]: Not in query params, but just in url. Anyway it should be fixed, I agree with u.

**Review by Nimond — APPROVED** [review]

**Review by lxmnk — APPROVED** [review]

**Pull request merged**